sdks/go/pkg/beam/runners/prism/internal/stage.go

```diff
@@ -315,7 +318,7 @@ progress:
 	for _, rr := range resp.GetResidualRoots() {
 		ba := rr.GetApplication()
 		if len(ba.GetElement()) == 0 {
-			slog.LogAttrs(context.TODO(), slog.LevelError, "returned empty residual application", slog.Any("bundle", rb))
+			j.Logger.LogAttrs(context.TODO(), slog.LevelError, "returned empty residual application", slog.Any("bundle", rb))
```

Comment:
Just as an idle comment: You can imagine how in some contexts, building up the associated logger, so that anything that you *always* want in the given debugging output (like bundle I'd) gets it's custom start, so it can be isolated.